src/App.js

```diff
@@ -38,6 +38,9 @@ function App() {
   return (
     <div className="App">
       <Tweet />
```

Comment:
this piece of code breaks your code. you need to remove it, its because you didnt pass a props to this tag, but inside Tweet.js you used props.